doc/source/whatsnew/v0.18.0.txt

```diff
@@ -235,6 +309,58 @@ Subtraction by ``Timedelta`` in a ``Series`` by a ``Timestamp`` works (:issue:`1
 
 .. ipython:: python
 
+   pd.NaT * 1
+   pd.NaT * 1.5
+   pd.NaT / 2
+   pd.NaT * np.nan
+
+``NaT`` defines more arithmetic operations with ``datetime64[ns]`` and ``timedelta64[ns]``.
```

Comment:
I think these additions came in after I merged with current master (otherwise there were conflicts). Should I remove these additions or leave them?